File: php/examples/post_services_cancel_request.php
<?php
require_once 'base_request.php';

// Example of POST /services/:id/cancel-request

$debug = true;
$service_id = $_ENV['SERVICE_ID'];

$request_data = '';
$request_params = '';
$url_path = '/services/' . $service_id . '/cancel-request';

send_request('POST', $url_path, $request_params, $request_data);

File: csharp/examples/PostServicesCancelRequest.cs
using System;
namespace Examples.examples
{
	// Example of POST /services/:service_id/cancel-request

	public class PostServicesCancelRequest : BaseRequest
	{
        private string _serviceId;

        public PostServicesCancelRequest()
        {
            _serviceId = Environment.GetEnvironmentVariable("SERVICE_ID");
            
            Console.WriteLine($"PostServicesCancelRequest: _serviceId={_serviceId}");
        }

        public override string SendRequest()
        {
            var body = Newtonsoft.Json.JsonConvert.SerializeObject( new {} );
            return PostRequest($"services/{_serviceId}/cancel-request", body);
        }
    }
}